juniper/__init__.py

```diff
@@ -14,4 +14,4 @@
     limitations under the License.
 """
 
-__version__ = '0.3.0'
+__version__ = '0.4.1'
```

Comment:
@pdiazv Is this version 0.4.0 or 0.4.1?

Reply:
This should be version 0.4.0, thanks for catching that discrepancy.